### PR TITLE
Include crcmod in cytomining Dockerfile

### LINKED ISSUE #49: Consider adding dependency `crcmod` to Docker images.
I noticed the warning below when using image `us.gcr.io/broad-dsde-methods/cytomining:0.0.3` with the cytomining workflow. Consider adding package `crcmod` to this Docker image, and any other Docker images used by these workflows with calls to `gsutil`. Installation instructions are shown via `gsutil help crcmod`.

```
WARNING: gsutil rsync uses hashes when modification time is not available at
both the source and destination. Your crcmod installation isn't using the
module's C extension, so checksumming will run very slowly. If this is your
first rsync since updating gsutil, this rsync can take significantly longer than
usual. For help installing the extension, please see "gsutil help crcmod".
```

### PR DESCRIPTION
**Pull request by sjfleming**
Closes #49 

This PR also bumps up the default docker image to the newest tag `0.0.4` that includes this change.

### CODE REVIEW

**Review by carmendv — APPROVED** [review]
Looks great!

File: pipelines/mining/build_docker.sh [hunk 1]
```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-# clone the latest repo for cytominer-databse and pycytominer
+# clone the latest repo for cytominer-database and pycytominer
 if [[ -d cytominer-database ]]; then rm -r cytominer-database; fi
 if [[ -d pycytominer ]]; then rm -r pycytominer; fi
 git clone https://github.com/cytomining/cytominer-database.git cytominer-database
```
[the remark this answers is not in the document]
Comment (sjfleming) [reply pipelines/mining/build_docker.sh:6]: Good point, I will do it that way

**Comment by sjfleming** [comment]
I don't know why GitHub says I "dismissed your review"... I actually acted on your review... here's the new version

**Review by carmendv — APPROVED** [review]
Looks great! Thanks :)

**Pull request merged**